openaerostruct/geometry/geometry_mesh_transformations.py

```diff
@@ -131,11 +131,16 @@ def initialize(self):
         """
         self.options.declare("val", desc="Initial value for chord lengths")
         self.options.declare("mesh_shape", desc="Tuple containing mesh shape (nx, ny).")
+        self.options.declare(
+            "gamma",
+            default=0.25,
+            desc="float in [0.,1.] which indicates the chord fraction where to do the chord scaling. 1. is trailing edge, 0. is leading edge",
```

Comment:
Could it still be valid if this value is not between 0 and 1? If not, add a check in `setup` that throws an error if gamma is < 0 or > 1

Reply:
Works for any float values